t/cli/test_ops.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+. ./t/cli/common.sh
+
+# test cli operations
+git checkout conf/config.yaml
+
+make init
+
+# apisix start - nginx.pid exists but no such process
+fakepid=9999
```

Comment:
In fact, there is already a test to cover it: https://github.com/apache/apisix/blob/5da97515cff8da8fa0f1c8ab9ee55662bd871f13/t/cli/test_main.sh#L698

Reply:
Thks, I'll drop these test cases.

Reply:
Reverted.